packages/auth/src/interaction/routes.ts

```diff
@@ -202,28 +199,48 @@ async function handleInteractionChoice(
       Buffer.from(config.identityServerSecret)
```

Comment:
should there be also similar changes for L120 and L125?

Reply:
I thought at first not to change those, as they're not being served by a GNAP endpoint. But now I think I'll make them uniform across the board.